crates/tools/src/main.rs

```diff
@@ -163,3 +160,11 @@ fn run(cmdline: &'static str, dir: &str) -> Result<()> {
     }
     Ok(())
 }
+
+fn run_rustfmt() -> Result<()> {
```

Comment:
It would be cool to have an `fn run_rustfmt_precommit`, which formats all files changed in git. See this for the proper git command invocation:

It would be cool to run rustup only on changed files (